Formula/rpm.rb

```diff
@@ -34,6 +34,13 @@ class Rpm < Formula
   depends_on "gettext"
   depends_on "xz"
   depends_on "ossp-uuid"
+  # use external libraries, until the internal have been fixed
+  depends_on "lua"
+  depends_on "syck"
+  depends_on "openssl"
+
+  # lots of OS X support things were missing from this release
+  patch :DATA
```

Comment:
In that case @afb we probably don't want to accept new rpm versions that require patching but would rather wait for an OS X-friendly stable release.